src/components/ComponentIndexPage/ComponentIndexListItem.js

```diff
@@ -61,10 +62,7 @@ const ComponentIndexListItem = React.memo(
       );
     } else {
       img = (
-        <img
-          src={placeholder}
-          alt={`Placeholder image for the ${name} component`}
-        />
+        <img src={placeholder} alt={`Placeholder for the ${name} component`} />
```

Comment:
just wondering is this formatting change intentional or was it based on personal settings?

Reply:
I should just be picking up _our_ prettier config, unless something changed with VS Code/prettier

Reply:
ok I see, strange that it was not formatted before for whatever reason